packages/url_launcher/url_launcher/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 5.7.4
+
+* Added a new `LaunchStatus` to handle the ActivityNotFound exception that `activity.startActivity` can throw.
```

Comment:
nit: Let's make this message about what have changed from the plugin's user perspective. e.g maybe something like: "Fixed a crash on Android when there is no activity to handle the URL intent, instead the launch method will now throw a Dart PlatformException." (assuming I got the consequences right, side note: would be helpful to describe the previous bad behavior and the new behavior in the PR description)